lib/calculator.rb

```diff
@@ -0,0 +1,53 @@
+class Calculator
+
+  attr_reader :total_sum, :dishes
+
+  def send_sms
+    answer = gets.chomp
+    if answer == "n"
+      puts "Please order again."
+      Menu.new.view_dishes
+  #  else
+#######  TWILIO HERE ###########
+    end
+  end
+
+  def error_check
+    fail "No such item on menu. Please re-order." unless verify_order
+  end
+
+  def shift_to_calculator
+    calculate()
+  end
+
+
+private
+
+  def calculate
```

Comment:
It feels like calculate has several responsiblities: calculating the total, calculating the sub totals, printing totals, checking for errors.  Could it be decomposed into several methods?